Breaking: Change license to GNU GPL version 3

soham4abc opened the pull request:
Fixes #19

soham4abc committed:
gpl3 added

soham4abc committed:
gpl3 added

markusguenther reviewed (changes requested):
Thank your @soham4abc for changing that. But we also list the License in the package.json and composer.json files ;)

It would be awesome if you can follow https://github.com/Unikka/neos-linkchecker#contribution

We use commitizen for the commit messages to have this with a general standard.
Thanks for taking care :)

soham4abc committed:
Update: GNU GENERAL PUBLIC LICENSE added (Fixes #19)

soham4abc commented:
@markusguenther  I have done the requested changes. If anything else is required , please do request!

markusguenther committed:
Breaking: Change license to GNU GPL version 3

As we are a neoscms package and neos uses GNU GPL we also need to use that license.

BREAKING CHANGE: License change with copy left

fix #19

markusguenther reviewed (approved):
Thanks for your contribution :)
